sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -419,8 +419,7 @@ bool _parse_options(int msg_type, gnrc_rpl_instance_t *inst, gnrc_rpl_opt_t *opt
                 dodag->dio_opts |= GNRC_RPL_REQ_DIO_OPT_PREFIX_INFO;
 #endif
                 gnrc_rpl_opt_prefix_info_t *pi = (gnrc_rpl_opt_prefix_info_t *) opt;
-                ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
-                if_id = gnrc_ipv6_netif_find_by_addr(NULL, &all_RPL_nodes);
+                if_id = gnrc_ipv6_netif_find_by_addr(NULL, &ipv6_addr_all_rpl_nodes);
```

Comment:
OT: 
I wonder what would happen if we use multiple interfaces running RPL concurrently? 
`ipv6_addr_all_rpl_nodes` would be equal on all the interfaces right?